vector/src/main/java/im/vector/app/features/html/EventHtmlRenderer.kt

```diff
@@ -63,14 +68,28 @@ class EventHtmlRenderer @Inject constructor(
                                 }
                     }
                 })
-                .usePlugin(MarkwonInlineParserPlugin.create())
                 .usePlugin(JLatexMathPlugin.create(44F) { builder ->
                     builder.inlinesEnabled(true)
                     builder.theme().inlinePadding(JLatexMathTheme.Padding.symmetric(24, 8))
                 })
     } else {
         builder
-    }.textSetter(PrecomputedFutureTextSetterCompat.create()).build()
+    }
+        .usePlugin(MarkwonInlineParserPlugin.create(MarkwonInlineParser.factoryBuilderNoDefaults()))
+        .usePlugin(object : AbstractMarkwonPlugin() {
+            override fun configure(registry: Registry) {
+                registry.require(MarkwonInlineParserPlugin::class.java, { plugin: MarkwonInlineParserPlugin ->
+                    plugin.factoryBuilder().addInlineProcessor(HtmlInlineProcessor())
+                })
+            }
+        })
+        .usePlugin(object : AbstractMarkwonPlugin() {
+            override fun configureParser(builder: Parser.Builder) {
+                builder.enabledBlockTypes(Collections.emptySet())
```

Comment:
with kotlin the idiomatic way to declare an empty set is with `kotlin.collections emptySet()`